File: 프로그래머스/0/120891. 369게임/369게임.js
function solution(order) {
    var answer = 0;
    let newArr = String(order).split("");
    for(let i=0; i<newArr.length; i++) {
        if(newArr[i] === "3" || newArr[i] === "6" || newArr[i] === "9") {
            answer++
        }
    }
    return answer;
}

File: 프로그래머스/0/120891. 369게임/369게임.java
class Solution {
    public int solution(int order) {
        int answer = 0;

        String str = order+"";

        for(int i=0; i<str.length(); i++){
            char c = str.charAt(i);
            if(c=='3'||c=='6'||c=='9') answer++;
        }

        return answer;
    }
}